environments/puppet/data/hosts/puppet.your.domain.yaml

```diff
@@ -61,5 +51,32 @@ rsyslog::udp_listen_address: '127.0.0.1'
 simp::classes:
   - 'simp::server'
 
-simp::server::classes:
-  - 'simp::puppetdb'
+### PuppetDB ###
+#
+# PuppetDB will not be enabled on your server unless you uncomment the
+# 'simp::server::classes' section below.
+
+# simp::server::classes:
```

Comment:
If the simp::server::classes already exists because simp config has configured other classes, uncommenting out these 2 lines is not going to be a good idea.  Instructions need to be more nuanced.